.travis.yml

```diff
@@ -3,85 +3,92 @@ sudo: required
-      curl -sSL https://github.com/libuv/libuv/archive/v1.14.0.tar.gz | sudo tar zxf - -C /usr/local/src
-      pushd /usr/local/src/libuv-1.14.0
-      sudo sh autogen.sh
-      sudo ./configure
-      sudo make
-      sudo make install
-      popd
-      sudo rm -rf /usr/local/src/libuv-1.14.0
-      sudo ldconfig
-    elif [ "$TRAVIS_OS_NAME" == "osx" ]; then
-      brew update
-
-      # Need to install openssl for .NET Core SDK 1.0 (legacy tests)
-      brew install openssl
-      mkdir -p /usr/local/lib
-      ln -s /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib /usr/local/lib/
-      ln -s /usr/local/opt/openssl/lib/libssl.1.0.0.dylib /usr/local/lib/
+- |
+  if [ "$TRAVIS_OS_NAME" == "linux" ]; then
+    sudo apt-get install make automake libtool
+    curl -sSL https://github.com/libuv/libuv/archive/v1.14.0.tar.gz | sudo tar zxf - -C /usr/local/src
+    pushd /usr/local/src/libuv-1.14.0
+    sudo sh autogen.sh
+    sudo ./configure
+    sudo make
+    sudo make install
+    popd
+    sudo rm -rf /usr/local/src/libuv-1.14.0
+    sudo ldconfig
+  elif [ "$TRAVIS_OS_NAME" == "osx" ]; then
+    brew update
 
-      brew install libuv
-    fi
+    # Need to install openssl for .NET Core SDK 1.0 (legacy tests)
+    brew install openssl
+    mkdir -p /usr/local/lib
+    ln -s /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib /usr/local/lib/
+    ln -s /usr/local/opt/openssl/lib/libssl.1.0.0.dylib /usr/local/lib/
 
-  - export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
-  - export PATH="$PWD/.dotnet:$PATH"
+    brew install libuv
+  fi
+- export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
+- export PATH="$PWD/.dotnet:$PATH"
+after_success: |
+  if [[ "$TRAVIS_OS_NAME" == "linux" && $TRAVIS_BRANCH == "master" && $TRAVIS_PULL_REQUEST == "false" ]]; then
+    echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ wheezy main" | sudo tee /etc/apt/sources.list.d/azure-cli.list
+    sudo apt-key adv --keyserver packages.microsoft.com --recv-keys 52E16F86FEE04B979B07E28DB02C46DF417A0893
+    sudo apt-get install apt-transport-https
+    sudo apt-get update && sudo apt-get install azure-cli
 
+    az storage blob upload-batch --destination "releases" --source artifacts/deployment --account-name roslynomnisharp --account-key $BLOB_KEY
+  fi
 notifications:
   slack:
     rooms:
-      - omnisharp:U358j4McaEOIzFqXvGexVokC#integrations
+    - omnisharp:U358j4McaEOIzFqXvGexVokC#integrations
     on_success: change
     on_failure: always
     on_pull_requests: false
-
 before_deploy:
-  - |
-    # Only deploy OSX-specific OmniSharp from the OSX.
-    # All other builds should be deployed by Linux CI.
-    if [ "$TRAVIS_OS_NAME" == "linux" ]; then
-      rm artifacts/package/omnisharp-osx.tar.gz
-    elif [ "$TRAVIS_OS_NAME" == "osx" ]; then
-      rm artifacts/package/mono.tar.gz
-      rm artifacts/package/omnisharp-linux-x86.tar.gz
-      rm artifacts/package/omnisharp-linux-x64.tar.gz
-    fi
-
+- |
+  # Only deploy OSX-specific OmniSharp from the OSX.
```

Comment:
Does the mac build produce different binaries? If so, should we disable cross-compile?